common-content/en/module/tools/installing-npm-dependencies/index.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "Installing dependencies with npm"
+headless = true
+time = 60
+facilitation = false
+emoji= "📚"
+objectives = [
+    "Use a dependency in a NodeJS program",
+]
++++
+
+To use a library, we need to fetch the code we're going to use. When using NodeJS, we use a tool called `npm` for this.
+
+First we need a `package.json` file - this a file that `npm` will read to understand your project.
+
+Make this `package.json` file in the same directory as your hyphen-counting program:
+
+```json
+{
+    "type": "module"
+}
+```
+
+The `package.json` contains a JSON object with information about your project. For now, we're just telling `npm` that our project is a module - this means we are allowed to use `import` in our program.
```

Comment:
They have already used npm - call back to that and have them make a bridge to prior learning here.